ddl/cluster.go

```diff
@@ -37,12 +37,15 @@ func (w *worker) onFlashbackCluster(d *ddlCtx, t *meta.Meta, job *model.Job) (ve
 			if j.ID == job.ID {
 				continue
 			}
+			// has ddl job in queue, return error
 			if !j.IsFinished() {
 				return true, errors.Errorf("Can't flashback cluster, other ddl jobs in ddl queue")
 			}
+			// job is finished in time range [flashbackTS, now], not support flashback now
```

Comment:
You mean: `no new DDL jobs(only DML) should be executed in [flashbackTS, now]`?

Reply:
Yes, not support it now.